Subject: [PATCH] extraversion: add eudyptula to version string
From: Nishad Kamdar <nishadkamdar@gmail.com>
This is Task 3 of the eudyptula challenge.
The version string is appended by eudyptula.

Signed-off-by: Nishad Kamdar <nishadkamdar@gmail.com>
---
diff --git a/Makefile b/Makefile
index 1c5d5d8c45e2..b2642e45586c 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 4
 PATCHLEVEL = 16
 SUBLEVEL = 7
-EXTRAVERSION =
+EXTRAVERSION = eudyptula
 NAME = Fearless Coyote
 
 # *DOCUMENTATION*
